tests/integration/test_charm.py

```diff
@@ -68,3 +75,85 @@ async def test_owasp_modsecurity_custom_rules(app_url_modsec_ignore: str):
     """
     response = requests.get(app_url_modsec_ignore)
     assert response.status_code == 200
+
+
+async def test_owasp_modsecurity_crs_relation(ops_test: OpsTest, app_name: str, tmp_path: Path):
+    """
+    arrange: given charm has been built and deployed
+    act: relate the ingress integrator with any-charm and use any-charm to toggle modsecurity
+        option through relation via ingress lib.
+    assert: modsecurity annotations should be attached and detached from the kubernetes ingress
+        resource according to the modsecurity option.
+    """
+    await ops_test.model.applications[app_name].set_config(
+        {"owasp-modsecurity-crs": "false", "owasp-modsecurity-custom-rules": ""}
+    )
+    any_charm_tmp_path = tmp_path / "any-charm"
```

Comment:
Can I assume this charm is not it charmhub yet and this piece is temporary? The charms can be uploaded but not listed in charhub, which is probably what we want here

Reply:
I think we should do that in early test stages.

Reply:
Early as in "before merging" or as in "in the near future"? If the later, could you add a ticket to our backlog so we will recall updating this?

Reply:
I will upload the `any-charm` before this being merged.